tests/nested/manual/fakeinstaller-real/task.yaml

```
@@ -228,7 +228,11 @@ execute: |
       remote.exec "sudo test -e /var/lib/snapd/device/fde/marker"
       remote.exec "sudo test -e /var/lib/snapd/device/fde/marker"
       remote.exec "sudo blkid /dev/disk/by-label/ubuntu-data-enc" | MATCH crypto_LUKS
-  
+
+      echo "Ensure recovery keys are available on classic FDE too"
+      remote.exec "sudo snap recovery --show-keys" > show-keys.out
+      MATCH 'recovery:\s+[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}-[0-9]{5}' < show-keys.out
```

This should work too:
```suggestion
      MATCH 'recovery:\s+([0-9]{5}-){7}-[0-9]{5}$' < show-keys.out
```